Amplify/Categories/DataStore/DataStoreCategory+HubPayloadEventName.swift

```diff
@@ -23,4 +23,15 @@ public extension HubPayload.EventName.DataStore {
     /// Dispatched when DataStore receives a sync response from the remote API via the API category. The Hub Payload
     /// will be a `MutationEvent` instance that caused the conditional save failed.
     static let conditionalSaveFailed = "DataStore.conditionalSaveFailed"
+
+    /// Dispatched on DataStore start and also every time a local mutation is enqueued and processed in the outbox
```

Comment:
Clarify wording: "Dispatched when: 1) the DataStore starts; 2) each time a local mutation is enqueued into the outbox; 3) each time a local mutation is finished processing."